config/betterquesting/DefaultQuests.json

```diff
@@ -3252,7 +3252,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "The §6Steam Roaster §rburns chemicals using steam. This machine will be used for the production of §6sulfur§r.",
+          "desc:8": "The §6Steam Roaster §rburns chemicals using steam. It\u0027s a reaction furnace. This machine will be used for the production of §6sulfur§r. The Roaster will be used in many chemical processes in LV and onwards.",
```

Comment:
I would personally write the second sentence as "The Roaster will **also** be used".